Dockerfile

```diff
@@ -38,6 +38,14 @@ RUN apt-get install -y --no-install-recommends \
               vim \
            && rm -rf /var/lib/apt/lists/*
 
+# install packages the are not needed
+# but are installed only to test the --disable-package
+# argument of autotools package
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+             libgd-dev \
```

Comment:
I am still not happy with this.

My view is that either we install the Debian libgd-dev AND have a test that checks that this library is not used in the compilation process. Or we do not install the libgd-dev.

Furthermore, as we do not do any checks for the other libraries ( which are avialable in Debian but should not be used by Octopus), this seems an odd one out.

My preference would be to remove installation of the `libgd-dev` here. 

I agree that it might be useful to check that we don accidentally link against system libraries. If you think this is worth checking, I would suggest that we have (i) a separate workflow (and container) for this, we (ii) do it more systematically for all the libraries that could be provided from Debian, and (iii) we automate it so that after the compilation we check that truly no Debian-provided libaries are linked, and that we can raise an error automatically should this happen anyway. Unless we do all of these things, there is no value in having parts of the test, because we wouldn't notice if something goes wrong.